lib/trento/application/integration/discovery/payloads/sap_system_discovery_payload.ex

```diff
@@ -276,11 +276,9 @@ defmodule Trento.Integration.Discovery.SapSystemDiscoveryPayload do
     @required_fields [
       :pid,
       :name,
-      :starttime,
```

Comment:
I don't really understand this. Why are you removing these fields? They should be always coming.
Do they have some empty field, and that's why ecto is discarding the messages?

What are the values that you are receiving in the failed message?

Reply:
Forn @abravosuse to trigger the failover scenario for me, he basically was stopping the primary instance; causing that the process information in the payload was something like:

```
"Processes" => [ % {
	"description" => "HDB Daemon",
	"dispstatus" => "SAPControl-GRAY",
	"name" => "hdbdaemon",
	"pid" => 26205,
	"textstatus" => "Stopped"
}],
```

As you see, that process is in status Stopped and doesn't include these fields.

Reply:
But this is already the elixir code. I would like to see the content of the real payload, to understand why is not casting those fields. Maybe there are not coming. And that's why having a test with the real payload would be ideal

Reply:
Thanks @rtorrero